components/gcp/container/component_sdk/python/kfp_component/google/common/_utils.py

```diff
@@ -120,3 +121,54 @@ def wait_operation_done(get_operation, wait_interval):
             ))
         return operation
 
+
+def with_retries(
+    func: Callable,
+    on_error: Callable[[], Any] = None,
+    errors: Exception = Exception,
```

Comment:
nit: Should this be `Tuple[Exception, ...]`?

Reply:
Done. Thanks.